app/konnect/configure/runtime-manager/runtime-groups/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Runtime Groups
+no_version: true
+---
+
+Konnect manages runtime configuration in runtime groups. Each runtime group acts
+as a separate control plane and can manage runtime configurations independently
+of any other group. You can find a list of all runtime groups in your organization
+on the [Runtime Manager overview](https://konnect.konghq.com/runtime-manager/).
+
+* A **runtime group** is a collection of API connectivity runtime instances
+sharing the same configuration and behavior space.
+
+* A **runtime instance** is a single data plane node with a single instance of
+a runtime, such as Kong Gateway. Runtime instances service traffic for the runtime
+group. All runtime instances in one runtime group
+must be of the same type. Currently, only Kong Gateway runtime types are supported.
+
+Every organization has one default runtime group. With an Enterprise subscription,
+you can configure two additional custom runtime groups in the same Konnect
+account to manage runtime instances and their configuration in any groupings
+you want.
+
+_[TBA: diagram]_
```

Comment:
Do we have a diagram of how runtime groups work somewhere, that I could repurpose/remake?

Reply:
None that I know of that fits directly here, but there are a few diagrams floating around that have RGs. 
What would be useful to diagram? CP>RG>DP relationship?
- CP = Konnect multi cluster SaaS
- RG = Isolated config scoping for clusters of DP
- DP = Gateway runtimes hosted by the user

Reply:
It can be derived from https://miro.com/app/board/o9J_l5CFZig=/?invite_link_id=900744065852

Reply:
Going to split this into a post phase 1 task. A digram might be useful, but it's not the top priority for the first iteration.